### ISSUE TITLE
letsencrypt-nginx-proxy-companion support?

### ISSUE THREAD

**Issue by fishfree** [MinimalCompact/thumbor#13]
https://github.com/JrCs/docker-letsencrypt-nginx-proxy-companion

**Comment by gingerlime** [comment]
Yes, you can use it together, since we already use nginx-proxy anyway. I don't have a recipe for it, but it should be relatively straightforward to add.

**Comment by gingerlime** [comment]
@fishfree I've added a recipe for docker-compose with letsencrypt-nginx-proxy-companion. Hope this helps :)

**Comment by fishfree** [comment]
@gingerlime Thank you very much! But I still failed. When I run docker logs ***, it showed:
2019/01/17 07:24:51 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/signal_le_service'

I guess I don't understand this comment "# NOTE: you must export your HOSTNAME for it to work" in the file recipes/docker-compose/letsencrypt/docker-compose.yml. Before running docker-compose up -d I run "HOSTNAME=thumbor.mycompany.com"

**Comment by gingerlime** [comment]
@fishfree you must either run `export HOSTNAME` or `export HOSTNAME={your.host.name}` before running `docker-compose up`. Or you can also edit the `docker-compose.yml` and replace `$HOSTNAME` with the fully qualified domain name. The hostname must resolve to the IP address of the host you're running on (that's how let's encrypt verifies the certificate for you).

**Comment by gingerlime** [comment]
Also I found that some times I had to restart everything for it to work. I'm not entirely sure why. Some times it just worked on the first `docker-compose up` (on a new IP/hostname/certificate), but some other times it didn't. Not sure why, and since it was hard to reproduce wasn't really able to investigate this.

**Comment by fishfree** [comment]
@gingerlime It works now, even after server rebooting. Thank you very much!

**Comment by gingerlime** [comment]
Great to hear @fishfree. Thanks for keeping me posted.
